app/src/main/java/org/schabi/newpipe/App.java

```diff
@@ -233,38 +232,34 @@ protected void initACRA() {
     }
 
     private void initNotificationChannels() {
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
-            return;
-        }
-
         String id = getString(R.string.notification_channel_id);
```

Comment:
Feel free to remove all of these `id` assignments and just inline the `getString()` call. It feels strange to reuse a variable this way, while inlining `getString()` is normal.